src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -336,28 +271,80 @@ class BaseCourseCard extends Component {
     return null;
   };
 
-  renderCourseStartDate = () => {
-    const { startDate, mode } = this.props;
+  renderOrganizationName = () => {
+    const { orgName, mode } = this.props;
+
     const isExecutiveEducation2UCourse = EXECUTIVE_EDUCATION_COURSE_MODES.includes(mode);
+    const execEdClass = isExecutiveEducation2UCourse ? 'text-light-300' : '';
+    const course = isExecutiveEducation2UCourse ? 'Executive Education' : 'Course';
+    const tooltipText = isExecutiveEducation2UCourse
+      ? 'Executive Education courses are instructor-led, cohort-based, and follow a set schedule.'
+      : 'Courses are on-demand, self-paced, and include asynchronous online discussion.';
+
+    if (orgName) {
+      return (
+        <p className={`mb-2 font-weight-light ${execEdClass}`}>
+          {orgName} &#x2022; {course}
```

Comment:
I think it would be better directly use conditional here instead of first defining a constant.